inventory-template.ini

```diff
@@ -1,19 +1,22 @@
 [OSv3:children]
 masters
 nodes
+etcd
 
 [OSv3:vars]
 ansible_ssh_user=root
 deployment_type=openshift-enterprise
 
 openshift_docker_additional_registries=${ADDITIONAL_INSECURE_REGISTRIES}
 openshift_docker_insecure_registries=${ADDITIONAL_INSECURE_REGISTRIES}
+openshift_docker_blocked_registries=registry.hacker.com
 
-openshift_disable_check=disk_availability,docker_storage,memory_availability
+openshift_disable_check=disk_availability,docker_storage,memory_availability,docker_image_availability
 
 openshift_hosted_router_create_certificate=true
 
 osm_use_cockpit=true
+openshift_use_manageiq=true
```

Comment:
I'm not sure we still need this. can you try installing without it and see if the admin gets created?